rundeckapp/grails-spa/packages/ui-trellis/src/app/components/common/ScheduledExecutionDetails.vue

```diff
@@ -0,0 +1,121 @@
+<template>
```

Comment:
It seems like this component is very similar to this one:
https://github.com/rundeck/rundeck/blob/main/rundeckapp/grails-spa/packages/ui-trellis/src/library/components/plugins/PluginDetails.vue

probably we should consider how to consolidate to one component

Reply:
I did see this while working on the other tab for plugins, but remember that there's some logic in the scheduledExecutionDetails that you asked me to remove it as it wasn't fully finished? I think it would be best to consolidate once it's clear that other piece won't be needed. Because if it is still needed, both components won't be so similar anymore.

Reply:
I looked into how those properties were used, I think they are basically not used. The PluginDetails.vue was the conversion of the original _description.gsp as it was used by the plugin properties gsps. It doesn't seem to support markdown, so since you added that feature here, let's keep this one, but we should merge them at some point and add the markdown capabilities